scripts/README.md

```diff
@@ -30,3 +30,10 @@ Used to create Pelorus helm based operator from the helm charts.
 
 Ensures modification to the operator files can be cleanly applied and allows
 to recreate operator easily.
+
+## scripts/send_example_payload.py
+
+Used for testing and demo purposes to send fake data to the webhook exporter.
+Usually the script is copied over, modified (APP_NAME, send_events()) and
```

Comment:
The change in send events would be in the function itself or just the intervals?

Reply:
Really not limited to anything, could be interval, etc... but don't want to make yet another application to do that so it's up to the person who could use that file as a base.

Reply:
I would let a comment about that it can customized to user's needs, but I would add CLI to change intervals and `APP_NAME`

Implementation example https://github.com/dora-metrics/pelorus/blob/master/scripts/update_projects_version.py#L370-L404